src/Utilities/RootSolvers/RootSolvers.jl

```diff
@@ -128,4 +132,18 @@ function find_zero(f::F, x0::T, ::NewtonsMethod,
   return x0, false
 end
 
+function find_zero(f::F, f′::F′, x0::T, ::NewtonsMethod,
+                   xatol=1e-3,
+                   maxiters=10_000) where {F, F′, T<:AbstractFloat}
```

Comment:
`maxiters` should be a more reasonable value. For saturation adjustment, we should not need more than 3. Generally, something like a 100 as a ceiling makes sense.

Reply:
These are just default values, which may be better fitted for testing correctness, we'll certainly prescribe 10-100 when called in saturation adjustment.